src/DumpCommand.php

```diff
@@ -81,12 +81,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         // find all commands
         $command = $script . ' list ' . $scriptOptions . ' --format=xml';
+        $command_exploded = [$script . ' list ' . $scriptOptions, '--format=xml'];
```

Comment:
This is still built as a single string but now it's in array. We need each part of the command as an array element

Reply:
patience. lol. I was fixing as you were revieweing